cluster/gce/manifests/cluster-autoscaler.manifest

```diff
@@ -17,7 +17,7 @@
         "containers": [
             {
                 "name": "cluster-autoscaler",
-                "image": "k8s.gcr.io/cluster-autoscaler:v1.12.0",
+                "image": "k8s.gcr.io/cluster-autoscaler:v1.13.0-rc.2",
```

Comment:
FYI, this version of 1.13 is still not built and tagged. is that alright?

Reply:
It is already released. 
see http://gcr.io/google-containers/cluster-autoscaler:v1.13.0-rc.2.

I also tried to pull the aliased k8s.gcr.io/cluster-autoscaler:v1.13.0-rc.2 and it worked